src/components/Reports/MinisterHousingAllowance/Steps/StepThree/Calculation.tsx

```diff
@@ -93,24 +94,27 @@ export const Calculation: React.FC<CalculationProps> = ({
   const { t } = useTranslation();
   const locale = useLocale();
 
-  const { handleNextStep } = useMinisterHousingAllowance();
+  const { handleNextStep, pageType, handleEditNextStep, setHasCalcValues } =
+    useMinisterHousingAllowance();
 
   const {
     values: { rentOrOwn },
   } = useFormikContext<FormValues>();
 
+  const isNew = pageType === PageEnum.New;
```

Comment:
Is the only time `isNew` false when the board has given feedback and requested action?

I think if that is the case, we could rename this variable as actionRequired or requestReturned